src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Agricultoras.kt

```diff
@@ -0,0 +1,19 @@
+package ar.edu.unahur.obj2.semillasAlViento
+
+class Agricultora(val parcelas: MutableList<Parcela>) {
```

Comment:
Las Parcelas no cambian 
```suggestion
class Agricultora(val parcelas: List<Parcela>) {
```